packages/calcite-components/src/components/table/table.tsx

```diff
@@ -68,6 +68,9 @@ export class Table implements LocalizedComponent, LoadableComponent, T9nComponen
   /** Specifies the layout of the component. */
   @Prop({ reflect: true }) layout: TableLayout = "auto";
 
+  /** When `true`, prevents focus and keyboard navigation of Table Header and Table Cell components. Selection affordances and slotted content within Table Cell components remains focusable. */
```

Comment:
The focus and keyboard navigation will still be present with AT active. WDYT about adding in some context for folks? So they get additional context that could help them decide if they'd like to disable it when AT is not active, without taking away support for AT users?

```suggestion
  /** When `true`, prevents focus and keyboard navigation of the `table-header` and `table-cell`s when assistive technologies are not active. Selection affordances and slotted content within `table-cell`s remain focusable. */
```

Reply:
Added

Reply:
@geospatialem I updated this to reflect the new property name and two values. Could you give a look over?